service/src/main/python/jcarbon/nvml/sampler.py

```diff
@@ -62,6 +63,11 @@ def sample(self):
                         'metadata': {'device': i},
                         'value': nvmlDeviceGetPowerUsage(handle) / 1000.0,
                     })
+                elif 'nvmlDeviceGetTemperature':
+                    sample['data'].append({
+                        'metadata': {'device': i},
+                        'value': nvmlDeviceGetTemperature(handle, NVML_TEMPERATURE_GPU),
```

Comment:
Where did `NVML_TEMPERATURE_GPU` come from? I don't see it defined anywhere in this file.

Reply:
This is the enum temperature sensor used as an argument. It worked when I imported it and also when I didn't somehow. Would it be clearer to define it as an import at the top?